tests/unit/test_progression_simulation.py

```
@@ -0,0 +1,128 @@
+"""
+Test script to simulate player progression from Level 1 to 50.
+"""
+import pytest
```

Ruff (0.8.2)

`pytest` imported but unused

Remove unused import: `pytest`

(F401)